src/components/pages/settings/ChannelSelection.tsx

```diff
@@ -144,38 +163,49 @@ export default function ChannelSelection({ emitable, submit }: IProps) {
       }
     }
 
-    const result = [].concat(
-      ...channels.map((channel: any) => {
+    const result = ([] as { channelId: string; channelName: string }[]).concat(
```

Comment:
can we Type this?

Reply:
Actually its IChannel type without _id. So, I replace it everywhere.